PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+# Model Name
+
+## Use-cases
+Model use cases
+
+## Description
+Description of model
+
+## Model
+
+ |Model        |Download  | Download (with sample test data)|ONNX version|Accuracy |
```

Comment:
Can you add a column `MD5 checksum` in the table for the checksum of the model, so users can check to see if a model file has been changed/updated since they last downloaded it

Reply:
And an `Opset version` column for the model

Reply:
done!